contracts/libs/Witnet.sol

```diff
@@ -355,7 +355,7 @@ library Witnet {
         ) {
             return sha256(_bytecode);
         } else {
-            return 0x0;
+            return keccak256(_bytecode);
```

Comment:
I think we can leverage the _De Morgan's laws_ to turn this into a positive expression:

```solidity
if (block.chainid == 1101 || block.chainid == 1442) {
    return keccak256(_bytecode);
} else {
    return sha256(_bytecode);
}
```

This is in fact not only more readable but also cheaper in gas terms, because the first comparison will short-circuit the `if` statement earlier than in the other approach.

Reply:
Moreover, we should work on a new WitnetRandomness implementation that relies on latest WitnetRequest implementation instead of WitnetMalleableBase, so we can deprecate (and remove from repository) the WitnetMalleableBase itself.

Reply:
Just created a new issue for this: 
- https://github.com/witnet/witnet-solidity-bridge/issues/364

Reply:
Thanks. All of that makes a lot of sense.